lib/module/plugin.js

```diff
@@ -8,6 +8,9 @@ import './middleware'
 export default function (ctx, inject) {
   // Options
   const options = <%= JSON.stringify(options.options) %>
+  <% if (typeof options.options.initialState === 'function') { %>
+  options.initialState = <%= serializeFunction(options.options.initialState).replace(/^\s+cov_[^\s]+;\s*$/mgi, '') %>
```

Comment:
The replace here is a bit ugly, but because jest coverage is true it inserts coverage tracking into the function which also gets serialized but those coverage tracking vars are ofc not available in the generated plugin

Reply:
never mind, should've just ignored it ofc ;)